figures/migrations/0015_add_enrollment_data_model.py

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.29 on 2020-12-09 14:17
+from __future__ import unicode_literals
+
+from django import VERSION as DJANGO_VERSION
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+import django.utils.timezone
+import model_utils.fields
+
+
+class Migration(migrations.Migration):
+
+    if DJANGO_VERSION[0:2] == (1,8):
+        dependencies = [
+            migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+            ('sites', '0001_initial'),
+            ('figures', '0014_add_indexes_to_daily_metrics'),
+        ]
+    else:  # Assuming 1.11+
+        dependencies = [
+            migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+            ('sites', '0002_alter_domain_unique'),
```

Comment:
This should work, no need to specify the latest migration from `sites`, so if you'd like it's possible to remove this `if` statement

```suggestion
            ('sites', '0001_initial'),
```